.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "submodules/brotli"]
```

Comment:
How did you solve the problem with brotli internal linking?

Reply:
I'm working on it, but now i just move `.brotli` to `git submodule`

In my opinion is a more correct solution and expandable